src/app/pipes/abi-method.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import {PipeTransform, Pipe} from '@angular/core';
+import {ABIDefinition} from 'web3/eth/abi';
+
+@Pipe({
+  name: 'abiMethod'
+})
+
+export class AbiMethodPipe implements PipeTransform {
+
+  transform(val: ABIDefinition): string {
+    if (!val) {
+      return null;
+    }
+    const inputs = val.inputs.map(input => input.name);
+    // const outputs = val.outputs.map(output => output.name + (output.name ? ' ' : '') + output.type);
```

Comment:
I personally would like to see types also, for both inputs and outputs, but not sure if that is best for general users.

Reply:
@jmank88 yeah, i've tried and it's getting too long and entangled